internal/fingerprint/fingerprint.go

```diff
@@ -33,16 +33,27 @@ var EXCLUDED_EXT = []string{
 	".toml", ".ttf", ".txt", ".utf-8", ".vim", ".wav", ".whl", ".woff", ".woff2", ".xht",
 	".xhtml", ".xls", ".xlsx", ".xpm", ".xsd", ".xul", ".yaml", ".yml", ".wfp",
 	".editorconfig", ".dotcover", ".pid", ".lcov", ".egg", ".manifest", ".cache", ".coverage", ".cover",
-	".gem", ".lst", ".pickle", ".pdb", ".gml", ".pot", ".plt",
+	".gem", ".lst", ".pickle", ".pdb", ".gml", ".pot", ".plt", "", ".pyi",
```

Comment:
The new patterns here excluded files without ending, such as "LICENSE" and files with file ending ".pyi"